### PR TITLE
Homework4: Yauheni Chachukha

### PR DESCRIPTION
**Pull request by chechuhae**

### CODE REVIEW

File: src/Cities.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+number_countries = int(input())
+dct_1 = {}
+for i in range(number_countries):
+    lst_1 = [str(el) for el in input().split()]
+    for symbol in lst_1[1::1]:
+        if dct_1.get(symbol) is None:
+            dct_1[symbol] = lst_1[0]
+        else:
+            # Check common cities. Example Brest in France and Belarus
```
Comment (belovmd) [inline src/Cities.py:36]: great note

File: src/Words.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+"""
+Слова
+Во входной строке записан текст. Словом считается последовательность
+непробельных символов идущих подряд, слова разделены одним или большим числом
+пробелов или символами конца строки. Определите, сколько различных слов
+содержится в этом тексте.
+"""
+
+str_1 = " asd£ 2 2    \n\n\n asd£  erer n\n\n !!232323\n\n    djkfj %%$£R(()) "
```
Comment (belovmd) [inline src/Words.py]: \+ \t \r

File: src/languages.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+French
+Выходные данные
+В первой строке выведите количество языков, которые знают все школьники.
+Начиная со второй строки - список таких языков. Затем - количество языков,
+которые знает хотя бы один школьник, на следующих строках - список таких
+языков.
+"""
+
+number_pupils = int(input())
+lst_1 = []
```
Comment (belovmd) [inline src/languages.py]: unclear naming here and below

**Review by belovmd — CHANGES_REQUESTED** [review]
Thank you for your work please take a look at next issues:

File: src/languages.py (hunk 3 above)
Comment (chechuhae) [reply src/languages.py]: Did all tasks

File: src/languages.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    for symbol in dct_of_languages:
+        if dct_of_languages.get(symbol) == number_pupils:
+            print(symbol)
+else:
+    print("У школьников нет общих языков")
+if count_one > 0:
+    print(count_one)
+    for symbol in dct_of_languages:
+        if dct_of_languages.get(symbol) != number_pupils:
+            print(symbol)
```
Comment (belovmd) [inline src/languages.py:54]: it's much more easier with set intersection and union

**Review by belovmd — COMMENTED** [review]
8,5

**Pull request closed without merging**